lxc/completion.go

```diff
@@ -108,6 +108,23 @@ func (g *cmdGlobal) cmpInstances(toComplete string) ([]string, cobra.ShellCompDi
 	return results, cmpDirectives
 }
 
+func (g *cmdGlobal) cmpInstanceNamesFromRemote(toComplete string) ([]string, cobra.ShellCompDirective) {
+	results := []string{}
+
+	resources, _ := g.ParseServers(toComplete)
+
+	if len(resources) > 0 {
+		resource := resources[0]
+
+		containers, _ := resource.server.GetInstanceNames("container")
+		results = append(results, containers...)
+		vms, _ := resource.server.GetInstanceNames("virtual-machine")
```

Comment:
I could be misremembering but I think `GetInstanceNames("")` will return all container and virtual machine names. That would be preferable to avoid two API calls here.

Reply:
That's correct, looks like ```InstanceTypeAny``` is defined as ```InstanceType("")```. Thanks for pointing that out!